app.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "Symfony Demo Application",
```

Comment:
I think we should use 4 spaces instead of a tab for indentation, don't we? Spaces indentation used everywhere in this project.